frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -331,17 +331,15 @@ export const NamespacedClusterServiceVersionTableRow = withFallback<
 
       {/* Provided APIs */}
       <TableData className={providedAPIsColumnClass}>
-        {!_.isEmpty(providedAPIs) ? (
-          _.take(providedAPIs, 4).map((desc) => (
-            <div key={referenceForProvidedAPI(desc)}>
-              <Link to={`${route}/${referenceForProvidedAPI(desc)}`} title={desc.name}>
-                {desc.displayName}
-              </Link>
-            </div>
-          ))
-        ) : (
-          <div> - </div>
-        )}
+        {!_.isEmpty(providedAPIs)
+          ? _.take(providedAPIs, 4).map((desc) => (
+              <div key={referenceForProvidedAPI(desc)}>
+                <Link to={`${route}/${referenceForProvidedAPI(desc)}`} title={desc.name}>
+                  {desc.displayName || _.startCase(desc.kind)}
```

Comment:
`_.startCase` can do some weird things to some names (e.g. `OAuthClient` -> `O Auth Client`). I'd avoid it and just show the kind as-is.

```suggestion
                  {desc.displayName || desc.kind}
```